substrate/frame/support/procedural/src/lib.rs

```diff
@@ -1097,8 +1097,11 @@ pub fn weight(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
-/// Compact encoding for arguments can be achieved via `#[pallet::compact]`. The function must
-/// return a `DispatchResultWithPostInfo` or `DispatchResult`.
+///
+/// ---
+///
+/// **Rust-Analyzer users**: See the documentation of the Rust item in
+/// `frame_support::pallet_macros::call`.
```

Comment:
Mistakes in this comment? 

This appears to not be deep linked, does it work for you?

Reply:
It was not supposed to be a link, I have added it though